lecture_1/homework/app/controllers/samurais_controller.rb

```diff
@@ -0,0 +1,56 @@
+class SamuraisController < ApplicationController
+    
+    def index
+        render json: clan.samurais.all.to_json(only: %i[id name armor battles start death])
+    end
+
+    def alive 
```

Comment:
Spoko pomysł, ale nie jest to restowa akcja :) Najładniej by to było zrobić w kolejnym zagnieżdżonym kontrolerze :D NP. Routes by wyglądał tak jak poniżej, a Twoje nierestowe akcje byłyby w nich akcją index :) W tym zadaniu wystarczylo można było to też zrealizować przez odpowiednie ify w akcji index w tym kontrolerze :)

```ruby
resources :clans, param: :name  do
  resources :samurais do
    resources :active, only: %i[index]
    resources :kaput, only: %i[index]
  end
end
```